### PR TITLE
V2 sdk changes re release for Segment Wrapper

### PR DESCRIPTION
**Pull request by sharekris**

### CODE REVIEW

File: Analytics/Integrations/Kahuna/SEGKahunaIntegration.m [hunk 1]
```diff
@@ -80,17 +62,22 @@ - (void)start
         // We just need one call to launchWithKey and not multiple. The "start" method is called
         // everytime the app comes to foreground.
         if ([SEGKahunaPushMonitor sharedInstance].kahunaInitialized == NO) {
-            [KahunaAnalytics launchWithKey:apiKey];
-
+            [self.kahunaClass setDebugMode:true];
```
Comment (TorreyKahuna) [inline Analytics/Integrations/Kahuna/SEGKahunaIntegration.m]: Shouldn't this be controlled by a setting from Segment?

Comment (sharekris) [inline Analytics/Integrations/Kahuna/SEGKahunaIntegration.m]: Just removed that line setDebugMode. It was added in by mistake. Segment iOS has a debug more setting, but there is no easy way to get it. The variable itself is a static variable in a implementation class that we cannot add in our wrapper layer.

File: Analytics/Integrations/Kahuna/SEGKahunaIntegration.h [hunk 2]
```diff
@@ -14,15 +14,17 @@
...
 @end
 
 
 @interface SEGKahunaPushMonitor : NSObject
-@property (atomic) NSDictionary *pushInfo;
-@property (atomic) UIApplicationState applicationState;
-@property (atomic) BOOL kahunaInitialized;
+@property (nonatomic) NSDictionary *pushInfo;
+@property (nonatomic) UIApplicationState applicationState;
+@property (nonatomic) BOOL kahunaInitialized;
```
Comment (rayleeriver) [inline Analytics/Integrations/Kahuna/SEGKahunaIntegration.h:25]: I vaguely remembered some of these properties were changes from nonatomic to atomic.  Why are we changing them back?

Comment (sharekris) [inline Analytics/Integrations/Kahuna/SEGKahunaIntegration.h:25]: You are right. This needs to be atomic. Thanks.

**Pull request closed without merging**